tests/Tokens/setInterestRateModelTest.js

```diff
@@ -22,38 +22,35 @@ describe('CToken', function () {
     });
 
     it("fails if called by non-admin", async () => {
-      expect(
-        await send(cToken, 'harnessSetInterestRateModelFresh', [newModel._address], {from: accounts[0]})
-      ).toHaveTokenFailure('UNAUTHORIZED', 'SET_INTEREST_RATE_MODEL_OWNER_CHECK');
+      await expect(send(cToken, 'harnessSetInterestRateModelFresh', [newModel._address], {from: accounts[0]}))
+        .rejects.toRevertWithCustomError('SetInterestRateModelOwnerCheck');
       expect(await call(cToken, 'interestRateModel')).toEqual(oldModel._address);
     });
 
     it("fails if market not fresh", async () => {
       expect(await send(cToken, 'harnessFastForward', [5])).toSucceed();
-      expect(
-        await send(cToken, 'harnessSetInterestRateModelFresh', [newModel._address])
-      ).toHaveTokenFailure('MARKET_NOT_FRESH', 'SET_INTEREST_RATE_MODEL_FRESH_CHECK');
+      await expect(send(cToken, 'harnessSetInterestRateModelFresh', [newModel._address]))
+        .rejects.toRevertWithCustomError('SetInterestRateModelFreshCheck');
       expect(await call(cToken, 'interestRateModel')).toEqual(oldModel._address);
     });
 
     it("reverts if passed a contract that doesn't implement isInterestRateModel", async () => {
-      await expect(
-        send(cToken, 'harnessSetInterestRateModelFresh', [cToken.underlying._address])
-      ).rejects.toRevert();
+      await expect(send(cToken, 'harnessSetInterestRateModelFresh', [cToken.underlying._address]))
```

Comment:
Codacy has a fix for the issue: [Replace `send(cToken,·'harnessSetInterestRateModelFresh',·[cToken.underlying._address]))` with `⏎········send(cToken,·"harnessSetInterestRateModelFresh",·[⏎··········cToken.underlying._address,`](https://app.codacy.com/gh/cloudwalk/compound-protocol/pullRequest?prid=9803552)
```suggestion
      await expect(
```